include/alpaka/acc/AccOmp5.hpp

```diff
@@ -73,8 +119,12 @@ namespace alpaka
             AtomicOmpBuiltIn     // thread atomics
         >,
         public math::MathStdLib,
-        public BlockSharedMemDynMember<>,
-        public BlockSharedMemStOmp5,
+#    if _OPENMP >= 201811
+        public detail::AccOmp5BlockSharedMem<OffloadBuiltInSharedMem>, // default value determined by CMake flag
+#    else
+          public detail::AccOmp5BlockSharedMem<OffloadBuiltInSharedMemOff>
+        ,
```

Comment:
That is an interesting formatting choice `clang-format` has made here...

Reply:
I will change the style of the base-class list to have leading commas instead of trailing, like here:
https://github.com/alpaka-group/alpaka/blob/develop/include/alpaka/acc/AccOacc.hpp#L64-L82

This way clang-format makes more sensible choices...

I also like this style better, because this way end-of-line comments make more sense and you only need to change one line when appending/removing at the end.